.github/workflows/test-playwright-and-deploy-to-fly-io.yml

```diff
@@ -2,10 +2,9 @@ name: Playwright Tests and Deploy to Fly.io
 on: push
 
 jobs:
-  e2e-tests:
-    name: Playwright test
+  playwright-tests:
+    name: Playwright Tests
     runs-on: ubuntu-latest # or macos-latest, windows-latest
-    # Update env variables with your own database credentials
```

Comment:
why was this removed?

Reply:
Reverted the deleted comment
[2309a75](https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/15/commits/2309a75ca762b3d5bf0ecbc7cfecf18baa5f05d8)